pandas/core/common.py

```diff
@@ -2672,6 +2672,9 @@ def is_list_like(arg):
      return (hasattr(arg, '__iter__') and
             not isinstance(arg, compat.string_and_binary_types))
 
+def is_named_tuple(arg):
+    return isinstance(arg, tuple) and hasattr(arg, '_fields')
```

Comment:
I had a look - while `namedtuple` is in half a dozen places, there's nowhere that checks its type.